test/itkMaskedAssignImageFilterGTest.cxx

```diff
@@ -136,7 +136,7 @@ TEST_F(MaskedAssignFixture, SetGetPrint)
   EXPECT_ANY_THROW(filter->GetConstant1());
 
   EXPECT_NO_THROW(filter->SetConstant1(1));
-  EXPECT_NO_THROW(filter->GetInput());
+  EXPECT_ANY_THROW(filter->GetInput());
```

Comment:
Looks like this case throws in debug but not release... This behavior comes from the Ternay ( and maybe binary ) base classes.